<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Current Sgtk Environment</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_start">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButton_pause">
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton_stop">
     <property name="text">
      <string>Stop / Create Log</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="context">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Your Current Context: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_time">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>65</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="logo_example">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/sg_logo.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <zorder>label_time</zorder>
    </widget>
   </item>
   <item row="4" column="1" rowspan="3">
    <widget class="QWidget" name="widget_2" native="true"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="../../../sgConfigs/default2test1/tk-multi-timeslogger/resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
